prestoadmin/util/slider.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Utility functions for dealing with slider
+"""
+
+
+def degarbage_json(garbage_output):
+    nest_level = 0
```

Comment:
After talking about this offline, we think we have an approach that doesn't require plucking the JSON out of a bunch of other output or cleaning up temp files on the slider master:
We're going to use -out, but we'll use /dev/stdout as the output file.
Things that don't work:
* `--out -`. slider doesn't follow the convention that a - for the output path means stdout
* process substitution `--out >(cat)`. Slider errors out with the following: `main.ServiceLauncher - java.io.FileNotFoundException: /dev/fd/63 (Permission denied)`

/dev/stdout seems to be sufficiently portable for our use:
http://unix.stackexchange.com/questions/36403/portability-of-dev-stdout